recipes/wcurl/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "wcurl" %}
+{% set version = "2024.07.10" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/curl/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 962bb72e36e6f6cedbd21c8ca3af50e7dadd587a49d2482ab3226e76cf6dcc97
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
It is just a shell script, right? I believe you can make it `noarch: generic` and, for Windows users that have a shell+curl, it should work too.

Reply:
I tried that in my original commit, but it didn't work: https://github.com/conda-forge/staged-recipes/runs/34096851717
Do I need to add a specific requirement for Windows? Where do I move the script to?

Reply:
It doesn't need to pass on Windows here. But, if you want to make sure it works, you can depend on m2-bash in the test phase. I would no bother with it though.

Reply:
Do you think this install script looks ok for Windows? I couldn't find an example of a package that installs in this same way to base my scripts on.
https://github.com/conda-forge/staged-recipes/blob/66ec60b89c84edaf9c6cb1454200303bba1f437e/recipes/wcurl/bld.bat

Reply:
I believe it would be `Library/usr/bin` but that would prevent us from making this noarch :-/

@isuruf, sorry for the ping, but do you have any advice for us here?

Reply:
I found that [`bat-extras`](https://github.com/conda-forge/bat-extras-feedstock/blob/main/recipe/meta.yaml) is similar in that it is also a noarch Bash script (several scripts, in this case). The main differences are:
- It has a `bash` dependency and a `__unix` dependency (I have no idea what this second one does).
- It uses a much more complicated [build script](https://github.com/eth-p/bat-extras/blob/master/build.sh) that I couldn't really decipher...